pkgs/applications/misc/wikidpad/default.nix

```diff
@@ -0,0 +1,76 @@
+# TODO@deliciouslytyped: It's not entirely clear how to use the Python infra here, however the current derivation works.
+{ lib
+, writeShellScript
+, fetchFromGitHub
+, stdenv
+, python3 # TODO@deliciouslytyped: This is overridden in all-packages to be python37 until upstream
+          # is fixed; python38 deprecates some functions WikidPad relies on, see: 
+          # https://github.com/WikidPad/WikidPad/issues/77
+, wrapGAppsHook
+, sqlite
+}: let
+  python = python.withPackages (p: with p; [ setuptools wxPython_4_1 six]);
+  src = fetchFromGitHub { inherit (lib.importJSON ./master.json) owner repo rev sha256; };
+
+  mainProgram = "WikidPad";
+  launcher = writeShellScript mainProgram ''
+    #https://stackoverflow.com/questions/59895/how-can-i-get-the-source-directory-of-a-bash-script-from-within-the-script-itsel
+    DIR="$( cd "$( dirname "''${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+    cd -- "$DIR/.." || { echo cd failed; exit; }
+    LD_LIBRARY_PATH="$LD_LIBRARY_PATH":"${lib.makeLibraryPath [ sqlite ]}" ${python.interpreter} WikidPad.py
+    '';
+
+in stdenv.mkDerivation {
+  pname = "wikidpad";
+  version = "master-2019-07-31";
+
+  inherit src;
+
+  nativeBuildInputs = [ wrapGAppsHook ];
+  propagatedBuildInputs = [ sqlite ]; # NOTE: wikidpad uses its own ctypes wrapper for sqlite, using cdll.
+                                      # This is also why we add it to LD_LIBRARY_PATH.
+  installPhase = ''
+    mkdir -p -- "$out"/bin
+    cp -r -- ${src}/. "$out"
+    cp -- ${launcher} "$out"/bin/${mainProgram}
+    '';
```

Comment:
```suggestion
  installPhase = ''
    mkdir -p $out/bin
    cp -r ${src}/. $out
    cp ${launcher} $out/bin/WikidPad
  '';
```

Reply:
Why? I guess it's technically redundant.